CONTRIBUTING.md

```diff
@@ -4,39 +4,39 @@ Thank you for your interest in contributing to the Azure DevOps Extension!
 
 There are many ways in which you can contribute, beyond writing code. The goal of this document is to provide a high-level overview of how you can get involved.
 
-# Have a question
+## Have a question
 
 Search existing github [issues](https://github.com/Microsoft/azure-devops-cli-extension/issues?q=is%3Aopen+is%3Aissue+label%3Aquestion) for similar questions first, and feel free to file an issue with a "question" tag to get our attention.
 
-# Found a bug
+## Found a bug
 
-#### Where to find known issues
+### Where to find known issues
 
 Check open [issues](https://github.com/Microsoft/azure-devops-cli-extension/issues?q=is%3Aopen+is%3Aissue+label%3Abug) before you file a new bug report.
 
 If you find your issue already exists, make relevant comments and add your reaction. Use a reaction in place of a "+1" comment:
 👍 - upvote
 👎 - downvote
 
-#### Report new issue
+### Report new issue
 
 The best way to get your bug fixed is to provide good repro steps. Use our [Bug template](https://github.com/Microsoft/azure-devops-cli-extension/issues/new/choose) to report any issues.
 
-# Feature requests
+## Feature requests
 
-#### Where to find existing requests
+### Where to find existing requests
 
 Check open [feature requests](https://github.com/Microsoft/azure-devops-cli-extension/issues?q=is%3Aissue+is%3Aopen+label%3AFeature)  before you file a new request.
 
 If you find your request already exists, make relevant comments and add your reaction. Use a reaction in place of a "+1" comment:
 👍 - upvote
 👎 - downvote
 
-#### Submit New Feature request
+### Submit New Feature request
 
 Use our Feature Request [template](https://github.com/Microsoft/azure-devops-cli-extension/issues/new/choose) to create a new feature request.
 
-# Get started
+## Get started
```

Comment:
All the content in this section is now at one level above that it should be. Please lower the heading hierarchy for all subitems in this section like Pull request etiquette etc.

Reply:
Do double check that out in other files as well. 
Due to pushing each top heading some  subitems have come to same level as the section heading.

Reply:
Double-checked